sentences/fr/vacuum_HassVacuumReturnToBase.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassVacuumReturnToBase:
+    data:
+      - sentences:
```

Comment:
Add simpler sentences like
- "Arrete l'aspirateur"
- "Stoppe l'aspirateur"

We decided to go with the `ReturnToBase` service instead of the `Stop` service because nobody really wants to let his/her vacuum die in the middle of the living room :) 

But the sentence should be dead simple.